modules/Bio/EnsEMBL/Compara/Utils/Test.pm

```diff
@@ -0,0 +1,100 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::Utils::Test
+
+=head1 DESCRIPTION
+
+Utility functions used in test scripts
+
+=cut
+
+package Bio::EnsEMBL::Compara::Utils::Test;
+
+use strict;
+use warnings;
+
+use Cwd;
+use File::Spec;
+use File::Basename qw/dirname/;
+
+
+=head2 get_repository_root
+
+  Description : Return the path to the root of the repository. Note that
+                this is constructed from the path to this module.
+
+=cut
+
+my $repository_root;
+sub get_repository_root {
+    return $repository_root if $repository_root;
+    my $file_dir = dirname(__FILE__);
+    my $original_dir = cwd();
+    chdir($file_dir);
+    my $cur_dir = cwd();
+    chdir($original_dir);
+    $repository_root = File::Spec->catdir($cur_dir, File::Spec->updir(), File::Spec->updir(), File::Spec->updir(), File::Spec->updir(), File::Spec->updir());
```

Comment:
I'm always a little bit afraid of this way of writing code as if anything changes it will break. What about the following?
```suggestion
($repository_root = $cur_dir) =~ s/ensembl_compara.*/ensembl_compara/;
```
I know it forces to have the repository under `ensembl_compara`, but who would dare to change it? :P

Reply:
This regex or a _while_ loop that traverses the directories up until it finds `ensembl-compara` ?

Reply:
Performance-wise I guess the former, but either could work if you think the latter will produce a more readable code.